src/systems/userinput/devices/daydream-controller.js

```diff
@@ -10,6 +13,15 @@ export class DaydreamControllerDevice {
     this.rayObjectRotation = new THREE.Quaternion();
     this.selector = `#player-${gamepad.hand}-controller`;
     this.pose = new Pose();
+    this.sittingToStandingMatrix = new THREE.Matrix4().makeTranslation(0, 1.6, 0);
+    navigator.getVRDisplays().then(d => {
+      const m = d[0].stageParameters.sittingToStandingTransform;
+      for (let i = 0; i < 16; i++) {
+        this.sittingToStandingMatrix.elements[i] = m[i];
+      }
+    });
+    this.matrix = new THREE.Matrix4();
```

Comment:
This are just caches and not actual state so they should be pulled out into consts or as a closure in `write` (slight preference for the former). This is both for style (so you know they are not actual persistent sate) and for controllers with multiple instances actually saves constructing another object (minor given how many exist)